test/run_tests.sh

```diff
@@ -185,9 +199,15 @@ function run_mp_op_tests {
 }
 
 function run_tests {
-  run_op_tests
-  if [[ "$XLA_SKIP_MP_OP_TESTS" != "1" ]]; then
-    run_mp_op_tests
+  # TODO: enable PyTorch/XLA tests on TPUVM first and will enable PyTorch tests and MP(multi processing) tests on TPUVM too in the future
+  if [[ "$PYTORCH_XLA_TESTS_ONLY" != "1" ]] ; then
```

Comment:
disable `run_op_tests_pt` because PyTorch has not yet been downloaded in `ci-tpu-test-trigger`;

disable `run_mp_op_tests` because of:
https://pantheon.corp.google.com/cloud-build/builds/573264e9-8a9d-4fae-9976-3df8891f7aa1?project=tpu-pytorch&jsmode=o&mods=allow_workbench_image_override